consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseConsentAPI.java

```diff
@@ -473,4 +473,18 @@ public Consent getByName(String name) throws UnknownIdentifierException {
         return consent;
     }
 
+    @Override
+    public Election retrieveElectionDul(Integer electionId, String consentId) throws UnknownIdentifierException{
+        Election election;
+        if (electionId != null) {
+            election = electionDAO.findElectionWithFinalVoteById(electionId);
+        } else {
+            election = electionDAO.getElectionWithFinalVoteByReferenceIdAndType(consentId, ElectionType.TRANSLATE_DUL.getValue());
+        }
+        if (election.getDataUseLetter() == null) {
```

Comment:
actually, i don't think you need this check at all now that we are returning the election. and the code that calls it checks whether the election is null